app/assets/javascripts/search.js

```diff
@@ -0,0 +1,101 @@
+;(function (global) {
+  'use strict'
+
+  var $ = global.jQuery
+
+  var SearchResultsContainer = (function () {
+    var $searchResultContainer
+    var $results
+
+    var _setSearchDataAttributesOnResults = function () {
+      /*
+        Add the lowercased search terms to a 'data-search' attribute
+        for each possible search result
+       */
+      $results.each(function () {
+        var searchTerms = $(this).find('h2').text()
+        $(this).attr('data-search', searchTerms.toLowerCase())
```

Comment:
I think it's more explicit to update the HTML, so I'm going to keep this the same if you're not fussed.